.gitignore

```diff
@@ -59,3 +59,19 @@ __pycache__/
 
 # android build
 nntrainer_for_android.tar.gz
+
+# windows build
+tools/nnstreamer-windows-resource/
+nntrainer/x64/
+nntrainer/Release/
+nntrainer/Debug/
+nntrainer/api/
+nntrainer/OpenBLAS-0.3.21-x86/
+nntrainer/iniparser/
+nntrainer/mman-win32/
+nntrainer/dirent/
+nntrainer/dlfcn-win32/
+.vs/
+Applications/MNIST/Release/
+Applications/MNIST/x64/
+Release/
```

Comment:
```Release/``` includes both ```blahblah/Release``` and ```Release```.
It's duplicated with ```nntrainer/Release```, too.

If you want to specify a directory, try, ```/nntrainer/Release``` instead.
Refer to https://www.atlassian.com/git/tutorials/saving-changes/gitignore